### DISCUSSION TITLE
how do i give it authorization?

### DISCUSSION THREAD

**Discussion by CameronTheUnlucky** [RogueEdit/onlineRogueEditor#108] [Q&A] [answered]
![Capture](https://github.com/RogueEdit/onlineRogueEditor/assets/161952164/7265d554-bf5e-4ed0-95e9-6333d206e9a3)
what do i do?

**Comment by JulianStiebler** [comment, marked as answer]
You can't. Its either the server lets us in or not. If he gives a 403 he blocked our requests and the server is in a secured mode.
This usually falls off after a timer or may not, but as of now it works again.
